client/modules/User/components/CollectionMetadata.jsx

```diff
@@ -117,7 +110,17 @@ function CollectionMetadata({ collectionId }) {
 }
 
 CollectionMetadata.propTypes = {
-  collectionId: PropTypes.string.isRequired
+  collection: PropTypes.shape({
+    id: PropTypes.string,
+    name: PropTypes.string,
+    slug: PropTypes.string,
+    description: PropTypes.string,
+    owner: PropTypes.shape({
+      username: PropTypes.string
+    }).isRequired,
+    items: PropTypes.arrayOf(PropTypes.shape({}))
+  }).isRequired,
+  isOwner: PropTypes.bool.isRequired
```

Comment:
@raclim it seemed like it made sense to pass down the `collection` object from the parent component since the parent already has the object.  But I find these prop types super annoying so idk, I may roll back some of the changes in the `CollectionMetadata` file and access the collection with the `useSelector`.